### PR TITLE
Move WIRELESS.cfg into the submodule + unzip overwrite

### PR DESCRIPTION
**Pull request by matinlotfali**
In this pull request, I am moving the `WIRELESS.cfg` into the `meta-mistysom` submodule.
I also allow overwriting the unzip command, as it allows us to run `./start.sh` multiple times.

### CODE REVIEW

File: Build/start.sh
Comment (mistyron) [inline Build/start.sh:28]: Good fix, I like it! Simplifies work within the container! :+1:

**Review by mistyron — APPROVED** [review]
LGTM

**Comment by matinlotfali** [comment]
I will merge after my `meta-mistysom` PR is merged.

**Pull request merged**
